src/modules/shared/pages/switch-context/switch-context.component.ts

```diff
@@ -76,6 +76,8 @@ export class PageSwitchContextComponent extends CoreComponent {
     }
 
     if (this.currentRole?.id !== role.id) {
+      sessionStorage.clear();
```

Comment:
Just thinking whether in future this might lead to bugs if somebody adds something to session storage and expects it to survive a context switch?  Maybe we should be more explicit about what needs clearing?

Reply:
As discussed if it becomes an issue we can either use session storage with role or we could be explicit with which keys should be cleared and clear only those.